.gitignore

```diff
@@ -9,3 +9,6 @@ build/*
 tools/scripts/github_token.json
 
 src/lib/graphql/schema.json
+# local dev
```

Comment:
if you add stuff that only exists for you, then add this to your local gitignore

Reply:
removed.